src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+const ADD_BOOK = 'bookstore/books/addBook';
+const REMOVE_BOOK = 'bookstore/books/removeBook';
+const lastId = 0;
+export default function addRemoveReducer(state = [], action) {
+  switch (action.type) {
+    // add action
+    case ADD_BOOK: return [
+      ...state, {
+        id: lastId + 1,
```

Comment:
- [ ] Is there really any need to implement the book `id` this way? I think with this implementation all books would have the id of 1 (`lastId + 1`) since with each dispatch `lastId` is initialized to 0, no? Please verify that.

  If you would not use my **optional** changes below, then refactor your `id` so that it is derived from the `action` like so: `action.id` and refactor the `addBook` action to expect an `id`

  **SUGGESTION:** We probably don't need this `lastId` here. In the next project, we can generate unique `id` for each book using the `UUID` package in the components and dispatch id with the title and author.

Reply:
This issue is well resolved! ✔️